data/transactions/logic/eval.go

```diff
@@ -3053,3 +3060,26 @@ func opAppParamsGet(cx *evalContext) {
 	cx.stack[last] = value
 	cx.stack = append(cx.stack, stackValue{Uint: exist})
 }
+
+func opLog(cx *evalContext) {
+	last := len(cx.stack) - 1
```

Comment:
Are there any checks to ensure the argument is bytes, not an integer? Relying on the assembler isn't enough, since it can't type check loading values from places like scratch space, global state, etc.